WordGame/Scenes/Home/HomeInteractor.swift

```diff
@@ -0,0 +1,132 @@
+//
+//  HomeInteractor.swift
+//  WordGame
+//
+//  Created by Kumari Ritu Pal on 26/07/22.
+//
+
+import Foundation
+import UIKit
+
```

Comment:
Just wanted to ask what setup are you using ?
I could see there is ViewModel from MVVM.
I could see Interactor & Presenter from VIPER.
or is it some other Architecture that you are following ?

Reply:
I have used clean architecture.